lib/git_reflow.rb

```diff
@@ -157,7 +155,11 @@ def deliver(options = {})
             say "There were problems commiting your feature... please check the errors above and try again.", :error
           end
         elsif !existing_pull_request.build_status.nil? and existing_pull_request.build_status != "success"
```

Comment:
I believe that the code should be:

`# Standard Rejection Messages for Error with Reflow Deliver
      def rejection_message
        if !build_status.nil? and build_status != "success"
          "#{build.description}: #{build.target_url}"
        elsif !approval_minimums_reached?
          "You need approval from at least #{self.class.num_lgtm} users!"
        elsif !all_comments_addressed?
          # Maybe add what the last comment is?
          "The last comment is holding up approval:\n#{last_comment}"
        elsif reviewers_pending_response.count > 0
          "You still need a LGTM from: #{reviewers_pending_response.join(', ')}"
        else
          "Your code has not been reviewed yet."
        end
      end

      def approval_minimums_reached?
        self.class.num_lgtm.length <= 0 or approvals.size >= self.class.num_lgtm.to_i
      end

      def all_comments_addressed?
        self.class.num_lgtm.length <= 0 or last_comment.match(self.class.lgtm_regex)
      end`

But I get the idea, refactoring the logic into pull_request.rb allows us to consolidate all the logic together per pull_request.